onadata/libs/authentication.py

```diff
@@ -157,7 +157,7 @@ def authenticate_credentials(self, key):
         try:
             if isinstance(key, bytes):
                 key = key.decode("utf-8")
-            token = self.model.objects.get(key=key)
+            token = self.model.objects.select_related("user").get(key=key)
```

Comment:
@DavisRayM could we pin it to the master DB?

Reply:
I'd suggest we don't; In the except part of this we retrieve from master if we fail to retrieve from the read replica which I feel is a bit better